Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM sbtscala/scala-sbt:openjdk-18.0.2.1_1.8.0_3.2.0
```

Comment:
Pretty arbitrary image, just found something quickly. The [repo describes itself as official but I dunno](https://github.com/sbt/docker-sbt).